src/payloads/send_location.rs

```diff
@@ -0,0 +1,40 @@
+// This file is auto generated by `cg` <https://github.com/teloxide/cg> (e634f65).
+// **DO NOT EDIT THIS FILE**,
+// edit `cg` instead.
+use serde::Serialize;
+
+use crate::types::{ChatId, Message, ReplyMarkup};
+
+impl_payload! {
+    /// Use this method to send point on the map. On success, the sent [`Message`] is returned.
+    ///
+    /// [`Message`]: crate::types::Message
+    #[derive(Debug, PartialEq, Clone, Serialize)]
+    pub SendLocation (SendLocationSetters) => Message {
+        required {
+            /// Unique identifier for the target chat or username of the target channel (in the format `@channelusername`)
+            pub chat_id: ChatId [into],
+            /// Latitude of the location
+            pub latitude: f64,
+            /// Longitude of the location
+            pub longitude: f64,
+            /// Period in seconds for which the location will be updated (see [Live Locations], should be between 60 and 86400.
+            ///
+            /// [Live Locations]: https://telegram.org/blog/live-locations
+            pub live_period: u32,
```

Comment:
`u32` -> `i32`

Reply:
Still https://github.com/teloxide/teloxide-core/pull/25#discussion_r523722147

Also
> should be between 60 and 86400.

Reply:
> should

Reply:
Why allowing negative numbers, if it should be in range `60..86400`? This will at least lead to unexpected behaviour